docs/tutorials/create_a_fips_docker_image.md

```diff
@@ -2,6 +2,19 @@
 
 > Requires at least Ubuntu Pro Client version 27.7
 
+To check the version of the Pro Client, please run:
+
+```bash
+pro version
+```
+
+If the version is less than 27.7, you can update the Pro client by running the following commands:
+
+```bash
+sudo apt-get update
+sudo apt-get install ubuntu-advantage-tools
+```
```

Comment:
We'll probably need something similar to this in many places. Maybe we can have a separate howtoguide on making sure ubuntu-advantage-tools is up to date and we can just link to that here?

Reply:
Good idea, I will create that doc